lib/code_corps/emails/forgot_password_email.ex

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Emails.ForgotPasswordEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.Emails.BaseEmail
+
+  def create(user) do
+    BaseEmail.create
+    |> to(user.email)
+    |> template(template_id(), {})
+  end
+
+  defp template_id, do: Application.get_env(:code_corps, :postmark_receipt_template)
```

Comment:
@JoshSmith @begedin this isn't right.  Wasn't sure what to do with the configuration for this template_id.  Just put random numbers in the config files?